www/modules/centreon-open-tickets/class/rule.php

```diff
@@ -62,31 +64,35 @@ protected function _setActivate($select, $val) {
         $this->_db->query($query);
     }
 
-    public function getAliasAndProviderId($rule_id) {
+    public function getAliasAndProviderId($rule_id)
+    {
         $result = array();
         if (is_null($rule_id)) {
             return $result;
         }
 
-        $DBRESULT = $this->_db->query("SELECT alias, provider_id FROM mod_open_tickets_rule WHERE rule_id = '" . $rule_id . "' LIMIT 1");
-        if (($row = $DBRESULT->fetch())) {
+        $dbResult = $this->_db->query(
+            "SELECT alias, provider_id FROM mod_open_tickets_rule WHERE rule_id = '" . $rule_id . "' LIMIT 1"
+        );
+        if (($row = $dbResult->fetch())) {
             $result['alias'] = $row['alias'];
             $result['provider_id'] = $row['provider_id'];
         }
 
         return $result;
     }
 
-    protected function loadProvider($rule_id, $provider_id, $widget_id, $uniq_id=null) {
+    protected function loadProvider($rule_id, $provider_id, $widget_id, $uniq_id = null)
+    {
         global $centreon_path, $register_providers;
 
         if (!is_null($this->_provider)) {
             return ;
         }
 
         $centreon_open_tickets_path = $centreon_path . 'www/modules/centreon-open-tickets/';
-        require_once $centreon_open_tickets_path . 'providers/register.php';
-        require_once $centreon_open_tickets_path . 'providers/Abstract/AbstractProvider.class.php';
+        include_once $centreon_open_tickets_path . 'providers/register.php';
```

Comment:
I'm not sure that replacing require with include is a good option, as if the file doesn't exist, we shouldn't continue the script